engine/src/vs_math.h

```diff
@@ -26,6 +26,7 @@
 #include "macosx_math.h"
 #include <math.h>
 #include <float.h>
+#include <cmath>
```

Comment:
Perfect! Based on the documentation I've read of c++11 and later standards, this does indeed look like the correct form/correct syntax of the header to include.